config/filter.d/nginx-modsecurity.conf

```diff
@@ -0,0 +1,33 @@
+# Fail2ban filter configuration for nginx :: ModSecurity-nginx (nginx-modsec)
+# used to ban hosts, that were failed through nginx by the rate of ModSecurity events 
+#
+# Author: Charlie Hsieh (charlietag)
+#
+# To use 'modsecurity' filter you should have `ngx_http_modsecurity_module.so`
+# and define `modsecurity` and `modsecurity_rules_file` as described in ModSecurity-nginx documentation
+# https://github.com/SpiderLabs/ModSecurity-nginx
+#
+# Example:
+#
+#   http {
+#     ...
+#     modsecurity on;
+#     ...
+#     # http, server, or location:
+#     location ... {
+#       modsecurity_rules_file /etc/my_modsecurity_rules.conf;
+#       ...
+#     }
+#     ...
+#   }
+#   ...
+#
+
+[Definition]
+
+failregex = ^\s*\[[a-z]+\] \d+#\d+: \*\d+ \[client <HOST>(:\d{1,5})?\] ModSecurity:\s+(?:\[(?:\w+ \"[^\"]*\"|[^\]]*)\]\s*)*Access denied with code [45]\d\d
```

Comment:
Hi @sebres ,
Thanks for you suggestion.   I'm using 0.9.7, installed via Yum (CentOS 7).
And I don't need IPv6 supported.
I think I should PR into 0.9.x (if filing a bugfix/enhancement against 0.9.x series, choose `master` branch)
[https://github.com/fail2ban/fail2ban/pull/2347](url)
Sorry if I'm wrong.

Reply:
Still again, is there really any log-lines containing `[client 192.0.2.1:55555]`, so I mean the part with port `:55555` before closing bracket?
Because neither your test-logs show that, nor I think it is good in long term (due to IPv6 support).
> And I don't need IPv6 supported.

It does not matter how your personal preferences are, because both PR's would arrive in a main stream, so would then affect people which may "need IPv6 supported".